webhooks-extension/cmd/extension/README.md

```diff
@@ -0,0 +1,9 @@
+# Extension
+
+## Endpoints
+```
+POST /webhooks     - create new webhook -> request body must contain name, namespace gitrepositoryurl, accesstoken, and pipeline.  It may contain serviceaccount, registrysecret, helmsecret, and repositorysecretname
```

Comment:
I'm not sure about this files existence or where it should be placed.  Also.... given the set of limitations (the simple pipeline), is it odd that we also list all possible options???  Maybe seek input from others.

Reply:
So, it's very limiting in that the only `pipeline` that you can use is the [simple-pipeline](https://github.com/pipeline-hotel/example-pipelines/blob/master/config/pipeline.yaml) mentioned in the README (which reminds me that I need install instructions telling the user how to install this pipeline).

However, all of the possible options are the options that the `simple-pipeline` allow you to specify (the POST request passes these to the pipeline). For the final product, this should all be documented with the `simple-pipeline`'s Pipeline definition, NOT in this extension repo. However, since this is all hardcoded right now, I think it should be documented.